enhancements/kata-containers/kata-containers-dev-preview.md

```diff
@@ -0,0 +1,259 @@
+---
+title: kata-containers-dev-preview
+authors:
+  - "@aadam"
+reviewers:
+  - "@mpatel"
+approvers:
+  - "@mpatel"
+creation-date: 2020-06-07
+last-updated: 2020-06-07
+status: implementable
+see-also:
+  - TBD
+replaces:
+  - TBD
+superseded-by:
+  - TBD
+---
+
+# Kata Containers
+
+## Summary
+
+Kata containers is an open source project developing a container runtime using virtual machines and providing the same look and feel as with vanilla containers.
```

Comment:
Add link to Kata Containers project for easier review.

Reply:
done